explainer.md

```diff
@@ -687,13 +687,18 @@ enum VRFrameOfReferenceType {
 
 interface VRFrameOfReference : VRCoordinateSystem {
   readonly attribute VRStageBounds? bounds;
+  attribute EventHandler onboundschange;
```

Comment:
Should `VRFrameOfReference` or `VRCoordinateSystem`  inherit from `EventTarget`?